src/stores/gameStore.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { GameStage, GameStore } from "./gameStore";
+
+describe("create a playerStore with default settings", () => {
+  it("should return playerMoney 0", () => {
+    const gameStore = new GameStore();
+
+    expect(gameStore.gameStage).toEqual(GameStage.Start);
+    expect(gameStore.playerBet).toEqual(0);
+    expect(gameStore.tieBet).toEqual(0);
+    expect(gameStore.bankerBet).toEqual(0);
+    expect(gameStore.winner).toEqual("");
+    expect(gameStore.chosenBetValue).toEqual(0);
+    expect(gameStore.time).toEqual(0);
+    expect(gameStore.timer).toEqual(undefined);
+  });
+});
+
+describe("gameStage methods should work", () => {
+  it("should set gameStage correctly", () => {
+    const gameStore = new GameStore();
+    gameStore.setGameStage(GameStage.InitialBet);
+
+    expect(gameStore.gameStage).toEqual(GameStage.InitialBet);
+  });
+});
+
+describe("time methods should work", () => {
+  const gameStore = new GameStore();
+  it("should set time correctly", () => {
+    gameStore.setTime(60);
+
+    expect(gameStore.time).toEqual(60);
+  });
+
+  it("should add to time correctly", () => {
+    gameStore.addToTime(1);
+
+    expect(gameStore.time).toEqual(61);
+  });
+
+  it("should start and stop timer correctly", () => {
+    jest.useFakeTimers();
+    jest.spyOn(global, "setInterval");
+
+    gameStore.startTimer();
+    expect(setInterval).toHaveBeenCalledTimes(1);
+    expect(setInterval).toHaveBeenLastCalledWith(expect.any(Function), 1000);
+
+    gameStore.stopTimer();
+    expect(gameStore.time).toEqual(0);
```

Comment:
`jest.useRealTimers()` should be presented in the end after using  `jest.useFakeTimers()`